### PR TITLE
add winurl support

### PR DESCRIPTION
**Pull request by idettman**
# Add support for win url

## Description
Prebid Server is being updated to create hb_winurl targeting variables that will be used to track events for mobile apps, AMP, and PG. To make this work we need to update the Prebid Universal Creative.

Updated `renderAd()` to:
* Resolve the 'BIDID' macro in `winurl` with the value of `winbidid`
* If a problem with this process but `hb_winurl` exists, `log` a warning and skip
* Add the URL to an invisible image similar to the **PBJS** `triggerpixel` function in [Prebid.js/src/utils.js](https://github.com/prebid/Prebid.js/blob/master/src/utils.js)

## Related
[Issue #70](https://github.com/prebid/prebid-universal-creative/issues/70)

### CODE REVIEW

**Comment by bretg** [comment]
Confirmed that Isaac is ready for review here -- @harpere

File: test/spec/renderingManager_spec.js [hunk 1]
```diff
@@ -89,15 +89,17 @@ describe('renderingManager', function() {
...
     it('should render mobile app creative', function() {
       const renderObject = newRenderingManager(mockWin, env);
       let ucTagData = {
         cacheHost: 'example.com',
         cachePath: '/path',
         uuid: '123',
-        size: '300x250'
+        size: '300x250',
+        winurl: 'https://prebid-server.rubiconproject.com/event?t=win&b=BIDID&f=i',
+        bidid: 'AAAA-BBBB-CCCC-DDDD'
```
Comment (harpere) [inline test/spec/renderingManager_spec.js]: shouldn't this be `winbidid`?

File: test/spec/renderingManager_spec.js [hunk 2]
```diff
@@ -117,7 +119,9 @@ describe('renderingManager', function() {
       const renderObject = newRenderingManager(mockWin, env);
       let ucTagData = {
         uuid: '123',
-        size: '300x250'
+        size: '300x250',
+        winurl: 'https://prebid-server.rubiconproject.com/event?t=win&b=BIDID&f=i',
+        bidid: 'AAAA-BBBB-CCCC-DDDD'
```
Comment (harpere) [inline test/spec/renderingManager_spec.js]: same, shouldn't this be `winbidid`?

File: test/spec/renderingManager_spec.js [hunk 3]
```diff
@@ -169,7 +173,9 @@ describe('renderingManager', function() {
         cachePath: '/path',
         uuid: '123',
         size: '300x250',
-        hbPb: '10.00'
+        hbPb: '10.00',
+        winurl: 'https://prebid-server.rubiconproject.com/event?t=win&b=BIDID&f=i',
+        bidid: 'AAAA-BBBB-CCCC-DDDD'
```
Comment (harpere) [inline test/spec/renderingManager_spec.js]: same, shouldn't this be `winbidid`?

File: test/spec/renderingManager_spec.js [hunk 4]
```diff
@@ -211,12 +217,14 @@ describe('renderingManager', function() {
...
         isCrossDomain: () => true
-      }
+      };
       const renderObject = newRenderingManager(mockWin, env);
       let ucTagData = {
         adId: '123',
         adServerDomain: 'mypub.com',
         pubUrl: 'http://example.com',
+        winurl: 'https://prebid-server.rubiconproject.com/event?t=win&b=BIDID&f=i',
+        bidid: 'AAAA-BBBB-CCCC-DDDD'
```
Comment (harpere) [inline test/spec/renderingManager_spec.js]: again, shouldn't this be `winbidid`?

**Pull request merged**